build-go.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+
+# globals
+releases=../../releases
+
+# init colors
+txtnon='\e[0m'    # color reset
+txtred='\e[0;31m' # Red
+txtgrn='\e[0;32m' # Green
+txtylw='\e[0;33m' # Yellow
+
+function fail() {
+	printf $txtred%s$txtnon\\n "$@"
+	exit 1
+}
+
+function warn() {
+	printf $txtylw%s$txtnon\\n "$@"
+}
+
+function notice() {
+	printf $txtgrn%s$txtnon\\n "$@"
+}
+
+# dep checks
+reqbins="jq zip go"
+for b in $reqbins
+do
+	if ! type $b > /dev/null
+	then
+		fail "must have '$b' installed"
```

Comment:
yeah... jq i normally install with npm, zip with my package manager, and go from source. It would be nice to just be able to download them in the script if needed, but this is really just code that we're going to use internally (at least for now)